### PR TITLE
✨ Add `test_download_file()` test for `Sharepoint` source

### PR DESCRIPTION
**Pull request by AnnaGerlich**
## Summary

- Added missing test for `Sharepoint.download_file()`

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [ ] updates docstrings for any new functions or function arguments (if appropriate)
- [ ] updates `CHANGELOG.md` with a summary of the changes

### CODE REVIEW

**Comment by Trymzet** [comment]
Actually I don't think it makes sense to do `from viadot.sources import SharepointCredentials`; I would only expect sources to be importable that way. I think importing credentials with `from viadot.sources.sharepoint import SharepointCredentials` is more intuitive.

File: tests/unit/test_sharepoint.py [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 
 
 def test_to_df(sharepoint_url):
     s = Sharepoint(config_key="sharepoint_prod")
     df = s.to_df(sharepoint_url)
     assert not df.empty
+
+
+def test_download_file(sharepoint_url):
+    s = Sharepoint(config_key="sharepoint_prod")
```
Comment (Trymzet) [inline tests/unit/test_sharepoint.py:13]: Let's find a way not to publicly declare we don't have a test environment. 😅 If there is actually no dev or test envs, maybe we can create a fixture for `Sharepoint` and then this config key can be taken from an env variable that we set just for testing, eg. `VIADOT_TEST_SHAREPOINT_CONFIG_KEY`?

**Review by Trymzet — CHANGES_REQUESTED** [review]

File: tests/unit/test_sharepoint.py (hunk 1 above)
Comment (AnnaGerlich) [reply tests/unit/test_sharepoint.py]: [✨ Added support for sharepoint_config_key](https://github.com/dyvenia/viadot/pull/608/commits/0d21f22d013bd32ad4801aabf6051cdf0a8ad7b3)

**Review by Trymzet — APPROVED** [review]
Looks good

**Comment by Trymzet** [comment]
Pls rebase @AnnaGerlich then I can merge

**Review by Trymzet — APPROVED** [review]

**Pull request merged**